Coordinator/Coordinator/View/MainViewController.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MainViewController.swift
+//  Coordinator
+//
+//  Created by Vladka on 28.12.2022.
+//
+
+import UIKit
+
+/// Главная страница приложения
```

Comment:
```suggestion
/// Главная страница приложения
```
У нас не веб у нас не страницы а экраны